src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,36 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+/**
+ * The Command class represents the command entered by user.
+ */
+public abstract class Command {
```

Comment:
I like your use of abstract class here to abstract the different types of commands!